manifests/init.pp

```diff
@@ -108,17 +108,19 @@
 #   Group to run puppet_webhook as
 #
 class puppetwebhook(
-  String $pkg_version,
-  Enum['gem', 'puppet_gem'] $pkg_provider,
-  Puppetwebhook::Serverconfig $server_cfg,
-  Puppetwebhook::Appconfig $app_cfg,
-  Stdlib::Absolutepath $binfile,
-  Stdlib::Absolutepath $r10k_path,
-  Stdlib::Absolutepath $envfile_path,
-  String $webhook_user,
-  String $webhook_group,
+  Hash $app_cfg = {'production' => {'protected' => true, 'user' => 'puppet', 'pass' => 'puppet', 'chatops' => false, 'default_branch' => 'production', 'ignore_environments' => [], 'allow_uppercase' => true, 'loglevel' => 'info'}},
```

Comment:
Format this to be more readable?

Reply:
Keeping the default value in one line permit to get in Puppet Strings generated file.
I prefer to get the value in generated docs rather than not.